rrtstar.py

```diff
@@ -0,0 +1,128 @@
+import sys, pygame
+import random
+import numpy as np
+
+class Block(pygame.sprite.Sprite):

```

Comment:
```suggestion
class Block(pygame.sprite.Sprite):

```
class和def都應添加註釋，請參考程式撰寫規範

Reply:
可以傳新版的程式撰寫規範給我嗎?我只有舊版的，不知道class和def怎麼註解 P是random產生的東西，至於她是甚麼意義不大，所以就隨便給個代號不過在這邊是指point